Looks like Travis failed due to a missing pod upgrade?

> It seems like you've changed the constraints of dependency `MotionTransitioning` inside your development pod `MaterialComponents`.
You should run `pod update MotionTransitioning` to apply changes you've made.

```
Installing MotionTransitioning (3.3.0)
Generating Pods project
Checking and Updating MDCCatalog for EarlGrey.
EarlGrey setup complete.
You can use the Test Target: MDCEarlGreyTests for EarlGrey testing.
Integrating client projects
Sending stats
Pod installation complete! There are 6 dependencies from the Podfile and 10 total pods installed.
Analyzing dependencies
Fetching podspec for `MaterialComponents` from `../../`
[!] Unable to satisfy the following requirements:
- `MotionTransitioning (= 3.1.0)` required by `Podfile.lock`
- `MotionTransitioning (~> 3.3)` required by `MaterialComponents/MaskedTransition (35.2.0)`
It seems like you've changed the constraints of dependency `MotionTransitioning` inside your development pod `MaterialComponents`.
You should run `pod update MotionTransitioning` to apply changes you've made.
Traceback (most recent call last):
  File "/Users/travis/build/material-components/material-components-ios/scripts/release/manage_pods.py", line 211, in <module>
    main()
  File "/Users/travis/build/material-components/material-components-ios/scripts/release/manage_pods.py", line 197, in main
    install_all_podfile_dirs(args.directory, args.fast_pod_install, BLACKLIST_DIRS)
  File "/Users/travis/build/material-components/material-components-ios/scripts/release/manage_pods.py", line 105, in install_all_podfile_dirs
    install_podfile_dir(d, fast_install)
  File "/Users/travis/build/material-components/material-components-ios/scripts/release/manage_pods.py", line 92, in install_podfile_dir
    subprocess.check_call(cmd)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['pod', 'install', '--project-directory=./demos/Bare']' returned non-zero exit status 1
The command "scripts/prep_all" exited with 1.
```

--- a/MaterialComponents.podspec
+++ b/MaterialComponents.podspec
@@ -299,7 +299,7 @@ Pod::Spec.new do |s|
     ss.public_header_files = "components/#{ss.base_name}/src/*.h"
     ss.source_files = "components/#{ss.base_name}/src/*.{h,m}", "components/#{ss.base_name}/src/private/*.{h,m}"
 
-    ss.dependency "MotionTransitioning", "~> 3.0"
+    ss.dependency "MotionTransitioning", "~> 3.3"
     ss.dependency "MotionAnimator", "~> 1.0"
     ss.dependency "MotionInterchange", "~> 1.0"
   end

--- a/components/MaskedTransition/src/MDCMaskedTransition.m
+++ b/components/MaskedTransition/src/MDCMaskedTransition.m
@@ -43,7 +43,7 @@ static inline CGFloat LengthOfVector(CGVector vector) {
   return (CGFloat)hypot(vector.dx, vector.dy);
 }
 
-@interface MDCMaskedTransition () <MDMTransitionWithPresentation, MDMTransitionWithFallback>
+@interface MDCMaskedTransition () <MDMTransitionWithPresentation, MDMTransitionWithFeasibility>
 @end
 
 @implementation MDCMaskedTransition {
@@ -59,8 +59,8 @@ static inline CGFloat LengthOfVector(CGVector vector) {
   return self;
 }
 
-- (id<MDMTransition>)fallbackTransitionWithContext:(__unused id<MDMTransitionContext>)context {
-  return _shouldSlideWhenCollapsed ? nil : self;
+- (BOOL)canPerformTransitionWithContext:(__unused id<MDMTransitionContext>)context {
+  return _shouldSlideWhenCollapsed ? NO : YES;
 }
 
 #pragma mark - MDMTransitionWithPresentation

